react/.storybook/preview.js

```diff
@@ -1,6 +1,7 @@
 import { addDecorator, addParameters } from '@storybook/react';
 import { withA11y } from '@storybook/addon-a11y';
 import { withInfo } from '@storybook/addon-info';
+import '../src/index.css';
```

Comment:
```suggestion
import '../src/index.css';
```
why is this import?

Reply:
That pulls in the global styling for storybook for things that should be imported only once. So things like font files, or box sizing, etc.